src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,63 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
```

Comment:
(I have no opinion really :))

Security bundle introduced `security.console.user_password_encoder_command` while webserver bundle did `web_server.command.server_run`; i took the latter as template.

So im in the middle here :) however i always thought `console` and `command` are sibling namespaces. Where console namespace is used for library stuff (`console.command_loader`, `console.error_listener`), and `command` for commands solely.